packages/jest-cli/src/run_jest.js

```diff
@@ -67,26 +68,75 @@ const getTestPaths = async (
   });
 };
 
+function formatWhyRunning(whyRunning) {
+  const whyRunningArray = [];
+  const fakeLogger = {
+    error(...args) {
+      whyRunningArray.push(util.format(...args));
+    },
+  };
+
+  whyRunning(fakeLogger);
+
+  return whyRunningArray
+    .join('\n')
+    .split('\n\n')
+    .filter(entry => {
+      if (entry.startsWith('There are') || !entry) {
+        return false;
+      }
+
+      return entry.split('\n').some(l => l.includes('this._execModule('));
+    })
+    .map(entry => {
+      const [title, ...lines] = entry.split('\n');
+
+      const entries = lines
+        .map(line => line.split(/\s+-\s+/))
+        .map(([file, line]) => ({file, line}));
+
+      return {
+        entries,
+        title: title.replace('# ', ''),
+      };
+    });
+}
+
 const processResults = (runResults, options) => {
-  const {outputFile} = options;
-  if (options.testResultsProcessor) {
+  const {
+    outputFile,
+    isJSON,
+    onComplete,
+    outputStream,
+    testResultsProcessor,
+    whyRunning,
+  } = options;
+
+  if (whyRunning) {
+    runResults.openHandles = formatWhyRunning(whyRunning);
+  } else {
+    runResults.openHandles = [];
```

Comment:
we can get rid of this check by assigning default value to `whyRunning`

Reply:
done

Reply:
meh, it gave me lots of flow issues